docs/prod-host-setup.md

```diff
@@ -274,17 +274,38 @@ echo "swap partitions present (Recommendation: no swap)" \
 
 #### Side channel attacks
 
-It is strongly recommended that users follow the
+For the purposes of this document we assume a workload that involves arbitrary
+code execution in a multi-tenant context where each Firecracker process
+corresponds a single tenant.
```

Comment:
nit: s/corresponds/corresponds to/

Although, I trust your English much more than what I trust mine :P

Reply:
haha my bad, thank you!